otterdog/eclipse-openbsw.jsonnet

```diff
@@ -34,6 +34,30 @@ orgs.newOrg('eclipse-openbsw') {
       gh_pages_build_type: "legacy",
       gh_pages_source_branch: "gh-pages",
       gh_pages_source_path: "/",
+      rulesets: [
+        orgs.newRepoRuleset('main') {
+          include_refs+: [
+            "~ALL",
```

Comment:
I dont think that this is a good idea.
That will prevent direct pushes to any branch, including gh-pages.

I would suggest to start with ~DEFAULT_BRANCH